website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -22,7 +22,20 @@ GraphQL has several advantages, such as self-documenting, having a strong typing
 
 ## dbt Semantic Layer GraphQL API
 
-The dbt Semantic Layer GraphQL API allows you to explore and query metrics and dimensions. Due to its self-documenting nature, you can explore the calls conveniently through the [schema explorer](https://semantic-layer.cloud.getdbt.com/api/graphql).
+The dbt Semantic Layer GraphQL API allows you to explore and query metrics and dimensions. Due to its self-documenting nature, you can explore the calls conveniently through a schema explorer. 
+
+The schema explorer URLs vary depending on your [deployment region](/docs/cloud/about-cloud/regions-ip-addresses). Use the following table to find the right link for your region:
+
+| Deployment type |	Schema explorer URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://semantic-layer.cloud.getdbt.com/api/graphql |
+| EMEA multi-tenant	|	https://semantic-layer.emea.dbt.com/api/graphql |
+| APAC multi-tenant	|	https://semantic-layer.au.dbt.com/api/graphql |
+| Single tenant | `https://semantic-layer.YOUR_ACCESS_URL.com/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
+| North America Cell 1	|	`https://semantic-layer.YOUR_ACCOUNT_PREFIX.us1.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
```

Comment:
I believe this should be `https://YOUR_ACCOUNT_PREFIX.semantic-layer.us1.dbt.com/api/graphql`. Also, I'm unsure if the ST one should also be flipped.

Reply:
thank you @DevonFulcher ! i've changed this now. it's different to the regional MT ones where `semantic-layer` comes before the URL? 

i don't have a ST instance to test but is the updated PR correct?